### PR TITLE
fix: Allow Node titles for LangChain nodes

### PR DESCRIPTION
**Pull request by OlegIvaniv**
Include Langchain nodes path when retrieving nodes display names

### CODE REVIEW

File: src/parser.ts [hunk 1]
```diff
@@ -4,11 +4,14 @@ import glob from "fast-glob";
...
 
 const NODES_DIR = path.resolve("packages", "nodes-base", "nodes");
+const LANGCHAIN_NODES_DIR = path.resolve("packages", "@n8n", "nodes-langchain", "nodes");
 
 async function getDisplayNames() {
   const [nodeFilepaths, versionDescriptionFilepaths] = await Promise.all([
     glob(path.resolve(NODES_DIR, "**", "*.node.ts")),
     glob(path.resolve(NODES_DIR, "**", "versionDescription.ts")),
+    glob(path.resolve(LANGCHAIN_NODES_DIR, "**", "*.node.ts")),
+    glob(path.resolve(LANGCHAIN_NODES_DIR, "**", "versionDescription.ts")),
```
Comment (ivov) [inline src/parser.ts:14]: We're destructuring the first two arrays from `Promise.all()` for the first two glob calls, so I'd expect these additions will be ignored?

**Review by ivov — CHANGES_REQUESTED** [review]

File: src/parser.ts (hunk 1 above)
Comment (OlegIvaniv) [reply src/parser.ts:14]: My bad; I originally just commented out the first two items and kept the LC ones for testing and didn't realize that it's being deconstructed. Fixed & thanks for catching it.

**Review by ivov — APPROVED** [review]

**Comment by netroy** [comment]
~still seems to be failing https://github.com/n8n-io/n8n/actions/runs/7503946939/job/20429848500~
Nevermind. Had to build the `dist` folder before tagging.

**Comment by ivov** [comment]
@netroy Can you please update this release doc if it's outdated?

https://github.com/n8n-io/validate-n8n-pull-request-title#release

**Pull request merged**